MAINTAINERS.yml

```diff
@@ -0,0 +1,384 @@
+# This file contains information on who maintains what. It is parsed by
+# get_maintainer.py.
+#
+# File format
+# ###########
+#
+# "Subsystem title" (the quotes are needed if the title has more than one word):
```

Comment:
we probably should use "area" instead of subsystem to avoid confusion with what have under "/subsys/*" and other areas"

Reply:
Yeah, area is a better name for documentation and stuff too. Changed to use that in the maintainers file and code.